packages/cycle-scripts-one-fits-all/template/src/components/counter.tsx

```diff
@@ -6,45 +6,42 @@ import { DriverSources, DriverSinks } from '../drivers';
 
 export interface State {
     count : number;
-}
-export type Reducer = (prev  State) => State | undefined;
+};
+const defaultState : State = {
+    count: 30
+};
+
+export type Reducer = (prev : State) => State | undefined;
 export type Sources = DriverSources & { onion : StateSource<State> };
 export type Sinks = DriverSinks & { onion : Stream<Reducer> };
 
-export function Page1(sources : Sources): Sinks {
+export function Counter(sources : Sources) : Sinks {
     const action$ : Stream<Reducer> = intent(sources.DOM);
     const vdom$ : Stream<VNode> = view(sources.onion.state$);
 
-    const touchSpeech$ = sources.DOM
-        .select('[data-action="speak"]')
-        .events('click')
-        .map(({ currentTarget }) => (currentTarget as Element).textContent)
-        .map(text => (typeof text === 'string' ? text : ''));
-
     const routes$ = sources.DOM
         .select('[data-action="navigate"]')
         .events('click')
         .mapTo('/p2');
 
     return {
         DOM: vdom$,
-        speech: touchSpeech$,
+        speech: xs.never(),
```

Comment:
Ah that was before the extractSinks. 
By the way I see that now returns ALL sinks even ones like time which are source only